devtools/gotemplatelinter/flags.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"flag"
+	"strings"
+)
+
+type Flags struct {
+	Path          string
+	RulesToIgnore []string
+}
+
+func ParseFlags() Flags {
+	var path string
+	flag.StringVar(&path, "path", "", "path to go templates htmls")
```

Comment:
I agree with you. But seems golang `flag` pkg does not work

```
expected:
  go run ./mylinter path --ignore-rule rule-1 --ignore-rule rule-2
got:
  go run ./mylinter --ignore-rule rule-1 --ignore-rule rule-2 path
```

Do you think bottom one is acceptable? IMO no, so I googled a bit and it led me to [this workaround](https://stackoverflow.com/questions/70133798/how-to-provide-the-command-line-args-first-and-then-the-flags-in-golang), which I am going to implement. Please advise if otherwise~

Reply:
no need to work around. You invoke the command in `command -flag1 flag1value -flag2 flagvalue2 arg1 arg2`